lib/credentials/remote_credentials.d.ts

```diff
@@ -0,0 +1,33 @@
+import {Credentials} from '../credentials';
+export class RemoteCredentials extends Credentials {
+		/**
+		 * Represents credentials received.
+		 * @param {object} options - Override the default (1s) timeout period.
+		 */
+		constructor(options?: RemoteCredentialsOptions);
+	}
+    interface RemoteCredentialsOptions {
+        httpOptions?: {
+            /**
+             * Timeout in milliseconds.
+             */
+            timeout?: number
+        }
+        maxRetries?: number
+    }
+export class ECSCredentials extends RemoteCredentials {
```

Comment:
Ah, I see that this was moved. Instead, it should remain in the same file as before in case someone is importing ECSCredentials directly as a submodule.